profitbricks/client.py

```diff
@@ -61,9 +64,9 @@ def list_datacenters(self, depth=1):
 
     def delete_datacenter(self, datacenter_id):
         """
-        Removes the server from your Datacenter.
+        Removes the Data center and all its components such as servers, nics, loadbalancers, volumes.
```

Comment:
* lower case "data"
* shouldn't NICs be capitalized since it is an abbreviation?